server/routes.js

```diff
@@ -0,0 +1,14 @@
+const router = require('express').Router();
```

Comment:
Is there a particular reason why you're using a router here?

Reply:
I'm defining the routes on the router here and exporting them.